spec/validators/unreserved_username_validator_spec.rb

```diff
@@ -113,7 +119,7 @@ def self.account(service, username); end
         end
 
         def stub_reserved_usernames(value)
-          allow(Setting).to receive(:[]).with('reserved_usernames').and_return(value)
+          value&.each { |str| Fabricate(:username_block, username: str, exact: true) }
```

Comment:
_(I see this is marked ready but described as WIP, so not sure on state ... please ignore this if not actually ready...)_

On this stubbing -- we could probably remove this `stub_` method entirely, and a) in the spot where it's passing in `nil` to represent no reserved, do a `delete_all` on new model (because, I think?, the seeds will have been loaded, and we want to simulate having none), b) in the spot where it does pass in some, just build two records with fabricators directly?

Reply:
I'm trying to keep the changes minimal. Seeds aren't loaded in tests as far as I know so `delete_all` wouldn't help.